PennMobile/Home/Cells/Calendar/CalendarEvent.swift

```diff
@@ -21,17 +21,17 @@ final class CalendarEvent {
     
     func getDateString(fullLength: Bool) -> String {
         let formatter = DateFormatter()
-        formatter.dateFormat = fullLength ? "EEEE, MMM. d" : "E. MMM. d"
+        formatter.dateFormat = "EEEE, MMMM d"
```

Comment:
Previously, we abbreviated the weekday name when `fullLength` is `true` in order to save space. Otherwise, it would read "Saturday, March 7 to Sunday March, 15", which would be too long to fit on one line. Was getting rid of the abbreviation intentional? And if so, how did you resolve the length issue.

Reply:
Resolved by allowing the dates to flow onto a second line